frontend/src/components/adminpage/admin.css

```diff
@@ -0,0 +1,349 @@
+.welcome-section {
```

Comment:
This whole file is just copy-pasted from the `admin-dashboard.css` and  Please remove all the unnecessary styles from here!! There is a lot of unnecessary code in here!